README.md

```diff
@@ -64,6 +64,10 @@ patches `require` + `autoload` to use these absolute paths, thereby avoiding hav
 
 Problem outlined in this [talk](https://www.youtube.com/watch?v=kwkbrOwLsZY)
 
+## Troubleshooting
+
+If you're experiencing problems with loading your application and you're unable to successfully run `Bootscale.regenerate`, delete the `tmp/bootscale` folder.
```

Comment:
I'm not sure I understand how it's supposed to help. Did that really fixed your issue?

Also I'm not very likely this would solves most issues, so maybe saying it like: `try deleting the ...`